supervisor/errata_utils.py

```diff
@@ -157,6 +157,38 @@ def get_erratum_for_link(link: str, full: bool = False) -> Erratum | FullErratum
     return get_erratum(erratum_id, full=full)
 
 
+class ErratumBuilds(BaseModel):
+    product_list: dict[str, dict[str, set[str]]]
```

Comment:
This is hard to figure out; I thought about suggesting a set of nested types ErratumBuildListingsByArch, but for the use case, I think we can keep this structure, but it needs an example. Also, using RootModel would be a bit more elegant here, since the `product_list` name is meaningless.

``` py
class ErratumBuildListings(RootModel):
    # Map variant and architecture to a set of subpackage names
    # we ship for that architecture
    # { 
    #     "AppStream": {
    #           "SRPMS": {"libtiff"}
    #           "aarch64": {"libtiff", "libtiff-devel", ...}
    #     }
    # }
    root: dict[str, dict[str, set[str]]]